pkgs/applications/audio/librespot/default.nix

```diff
@@ -16,7 +13,7 @@ rustPlatform.buildRustPackage rec {
     sha256 = "1sdbjv8w2mfpv82rx5iy4s532l1767vmlrg9d8khnvh8vrm2lshy";
```

Comment:
Why sha256 is not changed?

Reply:
I messed this up, this is now properly updated